nomad/local/e2e-tests.nomad

```diff
@@ -45,16 +51,19 @@ variable "schema_properties_table_name" {
 variable "aws_access_key_id" {
   type        = string
   description = "The aws access key id used to interact with AWS."
+  default     = "DUMMY_LOCAL_AWS_ACCESS_KEY_ID"
```

Comment:
This is the same pattern we follow in the other nomad files; i don't love it, but I have much bigger fish to fry right now.